data/json/npcs/hints.json

```
@@ -143,7 +143,7 @@
     "I hear there's strange big berries around, but why don't I hear first hand if they are filling for their size or not?",
     "I've heard of a gang called The Doctors.  You know, bank robbers wearing stethoscopes.  What are they trying to achieve?  I use paper money to start fires now.",
     "You can hole up on a roof if you need to.  Yeah, the rain will fall on your head, but it's harder for the dead to get you there.  Get a tent, a rollmat, a sleeping bag and you're set.",
-    "I thought about putting a bag on my dog to carry some of my stuff.  It didn't work, because it was a chihuahua, and it was eaten by a rottweiler.  Should have put some kevlar on it like those Z9.  Oh well…",
+    "I thought about putting a bag on my dog to carry some of my stuff.  It didn't work, because it was a chihuahua, and it was eaten by a Rottweiler.  Should have put some Kevlar on it like those Z9.  Oh well…",
```

```suggestion
    "I thought about putting a bag on my dog to carry some of my stuff.  It didn't work, because it was a chihuahua, and it was eaten by a rottweiler.  Should have put some Kevlar on it like those Z9.  Oh well…",
```
rottweiler isn't a proper noun